ir/instr.cpp

```diff
@@ -3603,23 +3613,55 @@ void ShuffleVector::rauw(const Value &what, Value &with) {
 }
 
 void ShuffleVector::print(ostream &os) const {
-  os << getName() << " = shufflevector " << *v1 << ", " << *v2;
-  for (auto m : mask)
-    os << ", " << m;
+  const char *str = nullptr;
+  switch(op) {
+  case LLVMIR_ShufVec:   str = "shufflevector"; break;
+  case SSSE3_PShufB128:  str = "ssse3.pshuf.b.128"; break;
```

Comment:
can we just a single pshuf? There's not need for all these AFAICT

Reply:
also, rather than implementing new phsuf, why not lower them into shufflevector in llvm2alive?

Reply:
plus a add a test for the semantics pls

Reply:
Test added.
pshufb states that the negative mask returns zero element unlike shufflevector, so I think it is slightly different from itl.